requirements_test.txt

```diff
@@ -1,6 +1,10 @@
 pytest==6.2.5
 pytest-asyncio==0.15.1
-pytest-cov==3.0.0
+pytest-cov==2.12.1
 pytest-timeout==1.4.2
 requests==2.26.0
 websocket-client==1.2.1
+boto3==1.17.106
```

Comment:
Why are we adding these requirements to the test requirements? Aren't they needed when installing the package in production?

Reply:
Oh, boto3 is used for connecting Minio S3, it's might because we need it in the next PR about S3.